### PR TITLE
Phone keypad problem added in C++

### PR DESCRIPTION
**Pull request by gayatri517**
Code for issue #225 added
Kindly review it.
Thank you

### CODE REVIEW

**Comment by gayatri517** [comment]
@OjusWiZard Kindly review my PR.
Thank you.

**Comment by gayatri517** [comment]
@rudrakshi99 @OjusWiZard please review my PR

**Comment by gayatri517** [comment]
@raksha009 @plazzy99 @OjusWiZard Please look at my PR

File: Code/C++/phone_keypad.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	{
+		out[j]=keypad[digit][k];
+		//filling the remaining part
+		generate_names(in,out,i+1,j+1);
+	}
+	return;
+}
+int main()
+{
+	char in[100];
```
Comment (raksha009) [inline Code/C++/phone_keypad.cpp:30]: Specify what is in ant out above the lines in which they are declared.

File: Code/C++/phone_keypad.cpp [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+QU
+QV
+RT
+RU
+RV
+ST
+SU
+SV 
+Time Complexity : O(4^n)
+Space Complexity : O(n) */
```
Comment (raksha009) [inline Code/C++/phone_keypad.cpp:63]: specify n

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Comment by gayatri517** [comment]
@raksha009 I have made the changes that were requested. Kindly review it

**Review by raksha009 — APPROVED** [review]

**Comment by Srishti013** [comment]
Hi,
I am a GSSOC'21 participant. I would like to solve this issue in python. Please assign it to me.

**Pull request merged**